core/src/test/scala/org/typelevel/paiges/PaigesTest.scala

```diff
@@ -1,7 +1,7 @@
 package org.typelevel.paiges
 
 import scala.util.Random
-import org.scalatest.FunSuite
+import org.scalatest.funsuite.AnyFunSuite
```

Comment:
why is this change needed?

Reply:
Because of deprecation warning in scalatest https://github.com/scalatest/scalatest/blob/b54539cd8057b2c8207e5390c2aef83b95756538/scalatest/src/main/scala/org/scalatest/FunSuite.scala#L22

Reply:
that's a bummer. `AnyFunSuite` seems like a worse API and I like using FunSuite.